File: Acme/Acme/Model/EmployeeActivity.cs
namespace Acme.Model
{
    public class EmployeeActivity
    {        
        public int EmployeeId { get; set; }
        public Employee? Employee { get; set; } 

        
        public int ActivityId { get; set; } 

        public string Comment { get; set; }

    }
}

File: Front-End/src/app/model/EmployeeActivity.ts

export class Employee {
    firstName : string;
    lastName : string;
    email : string;

    constructor(firstName : string, lastName : string, email : string) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
    }
}

export class EmployeeActivity {
    employee : Employee;
    activityId : string;
    comment : string;
    
    constructor(employee: Employee, activityId: string, comment: string) {
        this.employee = employee;
        this.activityId = activityId;
        this.comment = comment;
    }
}
